django_components/app_settings.py

```diff
@@ -17,5 +17,9 @@ def LIBRARIES(self):
     def TEMPLATE_CACHE_SIZE(self):
         return self.settings.setdefault("template_cache_size", 128)
 
+    @property
```

Comment:
@EmilStenstrom I added this to the app_settings, but I'm not sure what's the logic of picking an app vs. a global setting.

I see that in the Middleware you went for a global setting.